bin/generate_participant.py

```diff
@@ -1 +1,29 @@
-1
+from pathlib import Path
+Path.cwd()
+data = Path.cwd()
+data.absolute() # Prints absolute path
+data.glob("Sub_*") # Selects all directories named Sub_* In Python, the glob module is used to retrieve files/pathnames matching a specified pattern.
+list(data.glob("Sub_*")) # Lists directories
+subj = list(data.glob("Sub_*")) # Adds variable for list   
+    
+
+sub_str = [ str(e) for e in subj] # subjects as string, converts elements from windows path to string retaining list format
+
+    
+len(sub_str) # Equals amount of elements in substr list
+total_elements = len(sub_str) # Variable for while loop
+element = 0 # List elements start at 0
+
+while element <= total_elements-1: # total_elements = 4 in this instance
```

Comment:
Formatting: space around `-`

You are generating index with while loop here. There are use cases for it, such as when you're index is not a fixed number, and when you need the index info for multiple matched variables.
`while` loop is commonly used for loops that require state checking (for example, a list that kept changing length)
```python
sub_id = []
while sub_str:  # stop when sub_str is empty
    cur_sub = sub_str.pop()  # pop an item from the list
    # do some stuff here
    # save output
    sub_id.append(cur_sub)
```
By the way, there's a way to automatically generate indices for a list of items in `for` loop. 
Checkout `enumerate`
```python
for i, cur_sub in enumerate(sub_str):
    ...
```